### DISCUSSION TITLE
Brazilian Portuguese Translation

### DISCUSSION THREAD

**Discussion by ChristoPy** [vuejs-translations/guidelines#8] [Q&A] [answered]
I'm excited to introduce Vue 3 to my fellow Brazilians. So a Portuguese translation would be so great!

**Comment by GuiDevloper** [comment, marked as answer]
Hey @ChristoPy, hope you doing great there!

Brazil already has it's own VueJS community called [vuejs-br](https://github.com/vuejs-br), there we translated the [v2 docs](https://github.com/vuejs-br/br.vuejs.org) and was been working through the [v3 translation](https://github.com/vuejs-br/docs-next) that you can use to present to your friends :)

Being the recently launched full revamp, we still planning to soon put those works and efforts on the new version of docs, I've been fascinated by it's writing, tutorials, VitePress use, etc. myself 😁

You can give a help on that v3 repo, share and expect for news coming soon! 🚀

**Reply by micheldpcarlos** [reply]
Hey @GuiDevloper, any news on this translation?

It would be nice to have the BR version of Vue 3 docs 💚

Is there anything I can do to help?

**Reply by GuiDevloper** [reply]
Hey @micheldpcarlos!

Things became confuse starting when a lot of work was needed to unite friends to finish the translation of [old v3 docs](https://github.com/vuejs-br/docs-next) only to realize almost everything of that become deprecated with the great [new v3 docs](https://github.com/vuejs/docs), then forgetting to reunite energy and people to still work on this (something we been re-doing when finding time since the v2)
Also, even these [guidelines](https://github.com/vuejs-translations/guidelines#readme) (that were recommended) lasted some time more to be written 👀 

Anyway, aside from that, some work had been happening motivated by a famous and epic translator of pt-PT:
[Discussion and repo of portuguese translation](https://github.com/vuejs-translations/guidelines/discussions/26)

Personal notes:
In the current age we see things fastly taking "renovated paths".
1 year before we were reading big pages and simply rewriting/proofreading it in pt-BR.
Weeks before I were working with a docs-reader that trains GPT models to answer questions about the content, with answering in any language being just a detail. Used Vue3 docs as one of the examples:
[GuiDevloper/nullstack-markprompt](https://github.com/GuiDevloper/nullstack-markprompt)

And currently we have chats that can simply search on-the-fly whatever you questions about and build answers with a great understanding of that subject (took a look at something like the BingAI already?)
In other words, I love docs and looks like soon the writing/reading/translating of them will become something else (changing mainly how our energy is used at those tasks)

**Reply by micheldpcarlos** [reply]
Thanks for the update @GuiDevloper. I appreciate all the energy spent in bringing a translation to our community!
I'll check how I can help in pt-PT translations.

Personal notes:
It is incredible how LLMs are innovating quickly and changing how we work. While I still see value in having good docs written in our language so beginners can visit and follow a path, discover features, etc. I agree that our energy will be used differently, especially in creating or using translations.
